src/main/java/org/terasology/structureTemplates/internal/ui/ToolboxItemRenderer.java

```diff
@@ -89,11 +90,11 @@ public Vector2i getPreferredSize(ToolboxTreeValue value, Canvas canvas) {
         TextureRegion texture = getTexture(value);
         if (texture == null) {
             List<String> lines = TextLineBuilder.getLines(font, text, canvas.size().x);
-            return font.getSize(lines);
+            return JomlUtil.from(font.getSize(lines));
         } else {
             int iconWidth = marginLeft + texture.getWidth() + marginRight;
             List<String> lines = TextLineBuilder.getLines(font, text, canvas.size().x - iconWidth);
-            return font.getSize(lines).addX(iconWidth);
+            return JomlUtil.from(font.getSize(lines)).addX(iconWidth);
```

Comment:
Let's start doing more computations in JOML-world and less in TeraMath 😉 
```suggestion
            return JomlUtil.from(font.getSize(lines).add(iconWidth, 0));
```

Reply:
Fair. I pretty much just found that `JomlUtil` wrapper and started throwing it everywhere regardless of the usage, attempting to minimize change. I tried a wider refactor via IntelliJ before this, it went poorly ... 😁